tests/test_morphdb.py

```diff
@@ -0,0 +1,161 @@
+
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import morph_tool.morphdb as tested
+import numpy as np
+import pandas as pd
+from nose.tools import assert_raises, eq_
```

Comment:
Minor because we don't enforce linting on test files, but `numpy` and `eq_` are unused.
The file could be reformatted to be compliant to pep8.